event.go

```diff
@@ -1,28 +1,30 @@
 package main
 
-type Change struct {
-	Before string `json:"before"`
-	After  string `json:"after"`
-	Ref    string `json:"ref"`
-}
-
-type Project struct {
-	Name        string `json:"name"`
-	Description string `json:"description"`
-	URL         string `json:"url"`
-	SSHURL      string `json:"ssh_url"`
-	Namespace   string `json:"namespace"`
-}
-
```

Comment:
Is this an accidental removal? Its not mentioned in the description.

Reply:
They were merged into the `RepositoryUpdate` type as they were never referenced throughout the repo